src/demo/demo.js

```diff
@@ -116,7 +116,7 @@ class Demo extends Component {
   constructor(props) {
     super(props)
     this.state = {
-      token: false,
```

Comment:
OMG! Is this the cause of `false` token tests?

Reply:
Yeap, seems like it. 🙈

Reply:
By looking at the code, it seems that `async=false` is legacy from the time when the demo app was not a react component.

Reply:
OK, I'm going to remove all instances of the `async=false` query param as a clean up. Also will re-run the tests to be sure.